.ruby-version

```diff
@@ -0,0 +1 @@
+2.1.9
```

Comment:
Nothing inside of these 4 commits modifies Ruby code.  If a version bump is needed, this is the wrong place.